### PR TITLE
Updated style docs to reflect updates for adding new rules.

### PR DESCRIPTION
**Pull request by Garbee**
Can view section in question using [rawgit](https://rawgit.com/Garbee/devtools-docs/bug/448004/docs/elements-styles.html). Fixes [bug 448004](https://code.google.com/p/chromium/issues/detail?id=448004) in the Chromium tracker.

### CODE REVIEW

**Comment by heatheramahan** [comment]
@Garbee: Thanks!

**Comment by paulirish** [comment]
brand new icon too

File: docs/elements-styles.html [hunk 1]
```diff
@@ -143,9 +143,8 @@ <h3 id="styles_edit">Editing Styles</h3>
 
 <h3 id="styles_add_new">Adding New Rules and Properties</h3>
   <li>You can add a new style rule to be considered in addition to those found
-  in the stylesheets loaded by the page. Click the <b>Gear</b>
-  <span><img src="../images/gear.png"></span> button and
-  select the <nobr><b>New Style Rule</b></nobr> option. A new rule with an
+  in the stylesheets loaded by the page. Click the <b>New Style Rule</b>
+  <span><img src="../images/add-style-rule-icon.png"></span> icon located in the upper-right corder of the styles editor. A new rule with an
```
Comment (kdzwinel) [inline docs/elements-styles.html]: "corder"

**Comment by Garbee** [comment]
Typo fixed. (Clearly I need something better than vim/sublime text to write in.)

**Pull request merged**